astropy/io/votable/util.py

```diff
@@ -165,34 +203,31 @@ def is_frame_of_reference(x):
         else:
             points = p[:]
 
-        out = ','.join([numeric_or_range(x) for x in points])
+        out = ",".join([numeric_or_range(x) for x in points])
         length = len(points)
         if has_frame_of_reference:
             if frames is not None and p[-1] not in frames:
-                raise ValueError(
-                    f"'{p[-1]}' is not a valid frame of reference")
-            out += ';' + p[-1]
+                raise ValueError(f"{p[-1]} is not a valid frame of reference")
```

Comment:
Somehow `{p[-1]}` lost both the quotes and the `!r` ...

Reply:
Oops, I thought I had fully reverted the changes in https://github.com/astropy/astropy/pull/13959#discussion_r1012405328 as requested by @tomdonaldson in https://github.com/astropy/astropy/pull/13959#discussion_r1014907345; however, I missed that @nstarman had also removed the quotes. That has now been fixed.